src/roberts.cpp

```diff
@@ -12,3 +12,17 @@ void roberts_reference(const cv::Mat& src, cv::Mat& dst) {
             dst.at<uint32_t>(y, x) = dx * dx + dy * dy;
         }
 }
+
+void roberts_parallel(const cv::Mat& src, cv::Mat& dst) {
```

Comment:
Need to add to https://github.com/dkurt/cv_winter_camp_2021/blob/master/include/algo.hpp

Reply:
Yes, thank you, I added it. I just forgot to add it to the modified files in the git :)